x-pack/plugins/infra/public/utils/formatters/data.ts

```diff
@@ -71,6 +63,6 @@ const BASES = {
 export const createDataFormatter = (format: InfraWaffleMapDataFormat) => (val: number) => {
   const labels = LABELS[format];
   const base = BASES[format];
-  const power = getPowerFromValue(val, base, labels);
+  const power = Math.floor(Math.log(val) / Math.log(base));
```

Comment:
Since we only a have a limited number of labels, we should probably cap this using `Math.min(..., labels.length - 1)`. Otherwise we might run out of SI prefixes :wink: